cmd/root.go

```diff
@@ -47,7 +47,7 @@ backend implementation allowing users to create kubernetes nodes without running
 This allows users to schedule kubernetes workloads on nodes that aren't running Kubernetes.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println(kubeConfig)
-		f, err := vkubelet.New(nodeName, operatingSystem, kubeNamespace, kubeConfig, taint, provider, providerConfig)
```

Comment:
I would recommend to keep the taint in the first version to make the image backward compatible with the old helm chart.
That is - 
if disableTaint is specified in the command, do not add any taint.
if taint is specified in the command, use the old taint
if neither disableTaint nor taint is specified, use the taint defined in new environment variables.

Reply:
Related: we should move away from using `:latest` in the chart and version things for real.